apps/src/lib/cli.rs

```diff
@@ -814,14 +880,25 @@ pub mod args {
     impl Args for Intent {
         fn parse(matches: &ArgMatches) -> Self {
             let key = SIGNING_KEY.parse(matches);
-            let node_addr = NODE.parse(matches);
+            let node_addr = NODE_OPT.parse(matches);
```

Comment:
We also need to update these in the `def()` below to use the same arg (`NODE_OPT` and `TOPIC_OPT`).

Also, there's a handy option on `Arg` defitions `conflicts_with`/`conflicts_with_all`. I think it would be nice to use here. E.g.: 
```rust
NODE_OPT.def().about("The gossip node address.").conflicts_with(TO_STDOUT.get_name())

TO_STDOUT.def().about(...).conflicts_with_all(&[NODE_OPT.get_name(), TOPIC.get_name()])
```